junction/templates/proposals/detail/base.html

```diff
@@ -27,10 +27,8 @@
                 {{ proposal.title }}
             </h1>
             <span>
-            by:
-            <b>
-            {{ proposal.author.get_full_name }}
-            </b>
+                by: <b>{{ proposal.author }} ({{ proposal.author.get_full_name }})</b>
```

Comment:
Let's have it like `{{ proposal.author.username }}` to be more explicit and relying less on default string representation of `User` object.